plugins/repository-gcs/src/test/java/org/elasticsearch/repositories/gcs/GoogleCloudStorageBlobStoreRepositoryTests.java

```diff
@@ -205,7 +177,7 @@ public TestGoogleCloudStoragePlugin(Settings settings) {
 
         @Override
         protected GoogleCloudStorageService createStorageService() {
-            if (randomServerErrors) {
+            if (hasRandomServerErrors()) {
```

Comment:
This seems like needless complexity?
Can't we just always create a GCS client with these retry settings and save ourselves the `hasRandomServerErrors` (as well as maybe even the whole retry `Boolean` flag )?

Reply:
Ok